### DISCUSSION TITLE
抢票时间（bw）

### DISCUSSION THREAD

**Discussion by 1Firefly** [mikumifa/biliTickerBuy#302] [提问专区] [answered]
问一下，自动抢票的话是只需要设置完时间就行了，还是要设置完后点一次开始抢票？

**Comment by Seventhdoge** [comment, marked as answer]
设置完时间之后就不用管了，程序会在既定时间自动开始抢的，这个时候腾开双手开始在手机上双线作战吧

**Reply by mikumifa** [reply]
是这样的，放着挂机，手机也别闲着，脚本并不是100%

**Reply by Wang-Nainai** [reply]
老师 您好想问问 两个电脑登陆同个账号抢票会被B站判定异常吗？

**Reply by mikumifa** [reply]
不太清楚，建议趁现在，自己试一下

**Reply by Wang-Nainai** [reply]
谢谢大大 前两天纯手抢 4个人就抢到了1张BW的票，晚上见到有人在咸鱼挂您的项目 印象之前在Github见到过您的项目 明天联票 二轮BW BML门票靠您项目了( •̀ ω •́ )✧
